framework/src/play/templates/GroovyTemplate.java

```diff
@@ -521,7 +508,7 @@ public String __reverseWithCheck_absolute_false(String action) {
             return __reverseWithCheck(action, false);
         }
 
-        private String __reverseWithCheck(String action, boolean absolute) {
+        private static String __reverseWithCheck(String action, boolean absolute) {
```

Comment:
Why you made it static? It's private method, it cannot be used from outside. Please make it non-static back.

Reply:
Thanks for the feedback and having taken the time to test for multiple versions !
I made this method static because my editor automatically notifies me when a method could be declared static (as an utility method), which I did here because it does not use any instance field or method. 
No problem if you still prefer to make it non-static back !

Reply:
@tomparle Ok, I know that editor notification. I also used it many years ago.
Now I am pretty sure that `static` is a bad practice in most cases. For many reasons. The fact that the method doesn't use any fields is not the key factor.

Reply:
@asolntsev thanks for the feedback. I removed the static identifier from the method. I'd also be glad if you could share me some reasons of why it can be a bad practice, or give me a link about it !

Also, I just found out a regression that I did not see before, in the `#{form}` tag of the CRUD module (this module is separated from the framework, which is why I did not see it at first). This has been fixed.

I think it would also be neat to add the following kind of migration note about the removed method "_" in Groovy templates - even if I do not think it has been much used anywhere except from the framework itself (how do we add this to the future release notes ?) :
The shorthand method `_(String className)` to load a class in Groovy templates has been removed since the underscore is now a reserved keyword in JDK9 and later versions. Please use the method `__loadClass(String className)` instead.

Reply:
@tomparle will this restriction of JDK9 on the Groovy templates have a knock on effect on https://github.com/codeborne/play-fastergt?

Reply:
@tazmaniax indeed, I proposed some changes to the `form` and `types` tag implementation, see https://github.com/codeborne/play-fastergt/pull/1
As I do not use this tag implementation, I wish someone who does could test it in his app.

Reply:
@tomparle I'll test shortly, thx! Out of interest is there a reason why you don't use this rendering implementation as it would appear to be an improvement over the standard implementation?

Reply:
@tazmaniax First, thank you for make me discover this implementation I was not aware of. I guess it's the main raison I was not using it ! But if indeed this is an improvement I will try it for sure.
Another issue is that I have already customized heavily CRUD tags to support additional features, and I guess I would have to merge it with faster GT implementation which would require some work. Anyway, please let me know of the test results !

Reply:
I've also done some customisation of the CRUD tags and it would be interesting to maybe coalesce them into this project if they would be of value to others

Reply:
Good idea ! I will see that after I test to integrate them on my project, and tell you if I can add some features useful to others.